test/verify/check-system-info

```diff
@@ -559,33 +507,49 @@ revision\t: 2.3 (pvr 004e 1203)
         b.enter_page('/system/hwinfo')
         b.wait_in_text('#hwinfo-system-info-list .hwinfo-system-info-list-item:nth-of-type(2) .pf-c-description-list__group:nth-of-type(1) dd', "2x POWER9 (architected), altivec supported")
 
-        # Memory details should be shown for generic QEMU memory
-        # demidecode not available on certain images
+        # Memory details should be shown from our mocked DMI information from systemd's test files.
         if not m.ostree_image:
```

Comment:
fedora-coreos is a recent Fedora 35, that should now actually work?

Reply:
Hmm this was already conditionality before, I guess fedora-coreos didn't have dmidecode?

Reply:
Exactly. But now with this PR it doesn't need to any more, as it surely does have udev. I mean we should be able to entirely drop the `if not m.ostree_image:` condition and run this on coreos as well.